tinygrad/jit.py

```diff
@@ -22,7 +22,6 @@ def get_input_replace(jit_cache: List[JitItem], input_rawbuffers:List[Buffer]) -
     for i,a in enumerate(ji.rawbufs):
       if a in input_rawbuffers:
         input_replace[(j,i)] = input_rawbuffers.index(a)
-  assert len(set(input_replace.values())) == len(input_rawbuffers), "some input tensors not found"
```

Comment:
Why was this removed?

Reply:
This method is called in graphs which contain a part of jit cache which might not have all inputs in their slice. I moved this check to the jit, so the behaviour is the same as it was for a single graph/regular jit.